releases/v1.6.0.md

```diff
@@ -0,0 +1,13 @@
+## BladeRunnerJS @tagVersion@
+
+BladeRunnerJS @tagVersion@ contains full Java 8 support and new features.
+
+##### Java 8
+
+BRJS was previously built and compiled with Java 8 but distributed to run against Java 7+. This is the first release that targets Java 8. We advise users to upgrade their Java versions to 8 and to ensure prod servers are upgraded where necessary.
+
+### Enhancements
+
+- `br.AliasRegistryClass` format has been simplified and now allows inline alias registration.
```

Comment:
I'm unsure if this should be mentioned publicly, it's basically an internal implementation detail. I don't think it's harmful though.